test/kata4.humanCatDogYears.test.js

```diff
@@ -1,3 +1,11 @@
-const { humanCatDogYears } = require("../src");
+const { humanCatDogYears } = require('../src');
 
-// Look Ma, no handlebars!!!
+describe('humanCatDogYears', () => {
+	test('check number inputs', () => {
+		expect(humanCatDogYears(0)).toStrictEqual([0, 0, 0]);
+		expect(humanCatDogYears(1)).toStrictEqual([1, 15, 15]);
+		expect(humanCatDogYears(2)).toStrictEqual([2, 24, 24]);
+		expect(humanCatDogYears(3)).toStrictEqual([3, 28, 29]);
+		expect(humanCatDogYears(4)).toStrictEqual([4, 32, 34]);
+	});
```

Comment:
as previously mentioned, it would be better separate out each of these expects into their own test block with a descriptive test description for each